snforge_std/src/lib.cairo

```diff
@@ -30,26 +30,26 @@ use cheatcodes::ContractAddress;
 use cheatcodes::CheatSpan;
 use cheatcodes::test_address;
 use cheatcodes::test_selector;
-use cheatcodes::prank;
-use cheatcodes::prank_global;
-use cheatcodes::start_prank;
-use cheatcodes::stop_prank;
-use cheatcodes::stop_prank_global;
-use cheatcodes::roll;
-use cheatcodes::roll_global;
-use cheatcodes::start_roll;
-use cheatcodes::stop_roll;
-use cheatcodes::stop_roll_global;
-use cheatcodes::warp;
-use cheatcodes::warp_global;
-use cheatcodes::start_warp;
-use cheatcodes::stop_warp;
-use cheatcodes::stop_warp_global;
-use cheatcodes::elect;
-use cheatcodes::elect_global;
-use cheatcodes::start_elect;
-use cheatcodes::stop_elect;
-use cheatcodes::stop_elect_global;
+use cheatcodes::cheat_caller_address;
```

Comment:
If you don't want to update the docs here, there should be a follow up for this.
Please remember to update the docs and changelog

Reply:
Docs will be in last PR